test/e2e/framework/ci-operator.go

```diff
@@ -89,7 +89,7 @@ func newCiOperatorCommand(t *T) CiOperatorCommand {
 		GCSPushCredentialsFlag(t),
 	)
 	cmd.Env = append(cmd.Env, KubernetesClientEnv(t)...)
-	cmd.Env = append(cmd.Env, "PATH=$PATH:/usr/bin")
+	cmd.Env = append(cmd.Env, "PATH=$PATH:/usr/bin:/usr/local/bin")
```

Comment:
This is necessary to run the e2e tests locally on mac since the manifest tool has been added. It isn't specifically relevant to this change, but I think it is harmless to add.